third_party/beam/sdks/go/pkg/beam/runners/direct/gbk.go

```diff
@@ -18,37 +18,88 @@ package direct
 import (
```

Comment:
this file with all the changes have a test upstream somewhere?

Reply:
I added some tests in gbk_test.go